### ISSUE TITLE
Replying to an embed suppressed message suppresses embeds from the reply as well

### ISSUE THREAD

**Issue by 0x464e** [discordjs/discord.js#7818]
### Which package is this bug report for?

discord.js

### Issue description

So, if you suppress embeds from a message, and then reply to that message with `msg.reply()` embeds will also be suppressed from your reply message.  
This is obviously not the desired behavior.

In my sample code I send and suppress a message with the bot, but you can just as well manually suppress a message with your Discord client and then fetch that message and try to reply to it. The embed still wont work.

<details>
  <summary>Click to see an image of the unwanted behavior</summary>
  <img src="https://i.imgur.com/qssbotZ.png"/>
</details>

If it's hard to see what's going on in that image, the bot replies to its own message with an empty message (the empty message should contain an embed)   
Also, I have no idea why it worked once there in-between.

I'd be happy to **try** to fix this myself, but I would need some guidance as to where I should look for the issue/what I should try to look for.

### Code sample

```typescript
const { Intents, Client } = require('discord.js');
const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });

client.on('messageCreate', async msg => {
    if (msg.content === 'test')
    {
        let sent_msg = await msg.channel.send('a message');
        await sent_msg.suppressEmbeds();
        await sent_msg.reply({ embeds: [{ description: 'test'}] });
    }
});

client.login('token').catch(console.error);
```

### Package version

discord.js@13.6.0

### Node.js version

v17.4.0

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

No Partials

### Which gateway intents are you subscribing to?

Guilds, GuildMessages

**Comment by newracket** [comment]
I was able to reproduce this on discord.js version 13.6.0 but not on the newest development version. Could you try to reproduce it on the latest development release? It may have already been fixed.

**Comment by Jiralite** [comment]
This is indeed fixed in main and is planned to be fixed in 13.7 - here are the relevant pull requests:

- https://github.com/discordjs/discord.js/pull/7515
- https://github.com/discordjs/discord.js/pull/7560

**Comment by 0x464e** [comment]
Ah nice, good to hear it's fixed. I tried to search but I guess my search my queries insufficient.  
I'll close this issue.
